Rust Talk 2 - with Joel Marcey (#2091)
mainmatter/mainmatter.com: src/posts/2023-08-01-pitching-rust-to-decision-makers-with-joel-marcey.md

---
title: "Pitching Rust to decision-makers - with Joel Marcey"
authorHandle: marcoow
tags: [rust]
bio: "Marco Otte-Witte"
description:

>   "Joel Marcey and Marco Otte-Witte share insights on pitching Rust adoption to

Reviewer: ```suggestion
  "Joel Marcey and Marco Otte-Witte share talk about approaches for pitching Rust to
```

  decision-makers."
og:
  image: /assets/images/posts/2023-08-01-pitching-rust-to-decision-makers-with-joel-marcey/og-image.jpg
tagline: |

>   <p>Director of Technology at Rust Foundation Joel Marcey and Mainmatter’s Founder Marco Otte-Witte discuss how to pitch Rust adoption to decision makers, success stories of different companies including 1password, KDAB, Tweede golf, and Meta, and Joel’s role at the Foundation.</p>

Reviewer: ```suggestion
  <p>Director of Technology at Rust Foundation Joel Marcey and Mainmatter’s Founder Marco Otte-Witte discuss how to pitch Rust to decision makers, share success stories of different companies including 1password, KDAB, Tweede golf, and Meta, and talk about Joel’s role at the Foundation.</p>
```
